src/main/java/com/backbase/eo/testing/events/configuration/EventEmitterConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package com.backbase.eo.testing.events.configuration;
+
+import lombok.Getter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+@Getter
+@ConfigurationProperties("backbase.event-emitter")
```

Comment:
there are already some properties with this prefix
https://github.com/Backbase/event-emitter/blob/main/src/main/resources/application.yml#L6
do you need to set `@ConfigurationProperties("backbase.event-emitter.custom-header-pairs")` instead?

Reply:
```
backbase:
  event-emitter:
    topic-names: ""
```
These properties are not used in the code and just re-used here:
```
spring:
  application:
    name: event-emitter
  cloud:
    stream:
      default-binder: activemq
      bindings:
        consumeEvent-in-0:
          destination: ${backbase.event-emitter.topic-names}
```
I'm pretty sure that's done to simplify the configuration